quickwit/quickwit-datetime/src/date_time_parsing.rs

```diff
@@ -36,22 +36,24 @@ pub fn parse_date_time_str(
     date_time_str: &str,
     date_time_formats: &[DateTimeInputFormat],
 ) -> Result<TantivyDateTime, String> {
+    let trimmed_date_time_str = date_time_str.trim();
```

Comment:
```suggestion
    let trimmed_date_time_str = date_time_str.trim_ascii();
```

Reply:
It is probably minor, but triming unicode is considerably more expensive than trimming ascii. 
(It requires decoding utf-8, and check if each individual char is a whitespace or not. )

I'd feel safer if we restricted ourselves to ascii. It will just prevent us from trimming weird whitespace like the japanese "　".